src/client/shadows/dynamicshadows.cpp

```diff
@@ -96,11 +96,24 @@ DirectionalLight::DirectionalLight(const u32 shadowMapResolution,
 		farPlane(farValue), mapRes(shadowMapResolution), pos(position)
 {}
 
-void DirectionalLight::update_frustum(const Camera *cam, Client *client, bool force)
+void DirectionalLight::update_frustum(const Camera *cam, Client *client, bool force, float timeoftheday)
 {
 	if (dirty && !force)
 		return;
 
+	if (timeoftheday < 0.1 || timeoftheday > 0.9 || (timeoftheday > 0.4 && timeoftheday < 0.6)) {
```

Comment:
What if `set_sky` is used, where the sun is tilted? The shadows would still change noticeably during these time limits.